src/css/skins/seven.less

```diff
@@ -129,55 +121,63 @@
     /* Styles for play button on idle */
     .jw-display-icon-container {
         border-radius: 50%;
-        border: 1px solid #333;
+        width: 4.5em;
+        height: 4.5em;
+        margin: -2.25em auto 0;
+        
     }
     
     /* Rail/slider styles */
     .jw-rail {
-        background-color: #384154;
+        background-color: rgb(65, 74, 76);
         box-shadow: none;
     }
 
     .jw-buffer {
-        background-color: #666F82;
+        background-color: #757575;
+
     }
 
     .jw-progress {
         background: @active-color;
     }
 
     .jw-knob {
-        width: .6em;
-        height: .6em;
-        background-color: #fff;
-        box-shadow: 0px 0px 0px 1px rgba(0,0,0,1);
-        border-radius: 1em;
+          width: .7em;
+          height: .7em;
+          background-color: #fff;
+          border:1 px solid black;
```

Comment:
@robwalch it needs to have something to differentiate it from the rail color. Perhaps we make the background match inactive color?